service/routes.py

```diff
@@ -468,6 +474,10 @@ class CustomerCollection(Resource):
     #------------------------------------------------------------------
     # CREATE A NEW CUSTOMER
     #------------------------------------------------------------------
+    @api.doc('create_customers')
+    @api.response(400, 'The posted data was not valid')
```

Comment:
I think we just have to add in that a 409 is also possible due to username conflict